### PR TITLE
added Berlin, fixed config file logic

### PR DESCRIPTION
**Pull request by LianaHus**
Fixed https://github.com/ethereum/remix-project/issues/2469

### CODE REVIEW

File: libs/remix-ui/solidity-compiler/src/lib/compiler-container.tsx [hunk 1]
```diff
@@ -53,17 +66,27 @@ export const CompilerContainer = (props: CompilerContainerProps) => {
...
-    api.setAppParameter('configFilePath', "/compiler_config.json")
+    api.setAppParameter('configFilePath', "compiler_config.json")
+
+    if (state.useFileConfiguration) {
+      api.fileExists("compiler_config.json").then((exists) => {
+        if (!exists || state.useFileConfiguration ) createNewConfigFile()
+      })
+    }
+
+    api.setAppParameter('configFilePath', "compiler_config.json")
```
Comment (yann300) [inline libs/remix-ui/solidity-compiler/src/lib/compiler-container.tsx:77]: you probably don't need to do it 2 times (see line 69)

File: libs/remix-ui/solidity-compiler/src/lib/compiler-container.tsx [hunk 2]
```diff
@@ -53,17 +66,27 @@ export const CompilerContainer = (props: CompilerContainerProps) => {
...
     setShowFilePathInput(false)
   }, [workspaceName])
 
+  useEffect(() => {
+
```
Comment (yann300) [inline libs/remix-ui/solidity-compiler/src/lib/compiler-container.tsx:82]: linting

**Comment by ryestew** [comment]
In this PR - but maybe also in beta - if you create a new file with the path - it will make a new file.  This info needs to be updated in the docs. (note to self)

**Comment by ryestew** [comment]
and very important - are the matomo tags here?  Please make sure the button to open the advanced panel is added and how often people are using the .json file for compilation - and more too.

**Comment by LianaHus** [comment]
> Here's a screenshot from beta - which does not have anything to do with this PR...maybe... image
> 
> not the 2 files named lala.json - beta asked me if I wanted to create a file named that when I put that in the path text box. So it hadn't seen the current file there.
> 
> When you try to delete one of the files - the FE selects both - but only 1 gets deleted - but the remaining can't be selected.

@ioedeveloper

**Review by yann300 — APPROVED** [review]

**Pull request merged**